manage-data/ingest/upload-data-files.md

```diff
@@ -12,46 +12,65 @@ products:
+    :::{applies-item} stack: ga 9.0.0-9.1.8
 
-:::{image} /manage-data/images/serverless-file-uploader-UI.png
-:alt: File upload UI
-:screenshot:
-:::
+    1. Drag a file into the upload area or click **Select or drag and drop a file** to choose a file from your computer.
 
-Drag a file into the upload area or click **Select or drag and drop a file** to choose a file from your computer.
+        You can upload different file formats for analysis with the Data Visualizer.
 
-You can upload different file formats for analysis with the Data Visualizer:
+        The supported file formats are:
+          * CSV, TSV, NDJSON, log files (up to 500 MB)
+          * PDF, TXT, RTF, ODF, Microsoft Office files (up to 60 MB)
 
-File formats supported up to 500 MB:
+        After you upload a file, you can inspect its data and make any necessary changes before importing it.
 
-* CSV
-* TSV
-* NDJSON
-* Log files
+    2. Click **Import**.
 
-File formats supported up to 60 MB:
+    3. Enter a name for the index where the data will be stored, then click **Import** again to complete the process.
 
-* PDF
-* Microsoft Office files (Word, Excel, PowerPoint)
-* Plain Text (TXT)
-* Rich Text (RTF)
-* Open Document Format (ODF)
+    :::
 
-The Data Visualizer displays the first 1000 rows of the file. You can inspect the data and make any necessary changes before importing it. Click **Import** continue the process.
+    :::{applies-item} stack: ga 9.2
 
-This process will create an index and import the data into {{es}}. Once your data is in {{es}}, you can start exploring it, see [Explore and analyze](/explore-analyze/index.md) for more information.
+    1. Choose whether to import the data to a new index or to an existing one.
+    2. Enter a name for the new index or select an existing index from the dropdown.
+    3. Drag one or more files into the upload area or click **Select or drag and drop a file** to choose files from your computer.
+
+       The supported file formats are PDF, TXT, CSV, log files and NDJSON.
+
+       After you upload your files, you can inspect the data and make any necessary changes before importing it.
+
+    4. Click **Import** to complete the process.
+
+    :::
+
+    :::{applies-item} serverless: ga
+
+    1. Choose whether to import the data to a new index or to an existing one.
+    2. Enter a name for the new index or select an existing index from the dropdown.
+    3. Drag one or more files into the upload area or click **Select or drag and drop a file** to choose files from your computer.
+
+       The supported file formats are PDF, TXT, CSV, log files and NDJSON.
+
+       After you upload your files, you can inspect the data and make any necessary changes before importing it.
+
+    4. Click **Import** to complete the process.
+
+    :::
```

Comment:
As Stack 9.2 and Serverless are identical, can we combine these into a single tab instead of two? I've seen this pattern elsewhere, [for example](https://github.com/search?q=repo%3Aelastic%2Fdocs-content%20%22stack%22%3A%20%22ga%209.3%22%2C%20%22serverless%22%3A%20%22ga%22&type=code):

* https://www.elastic.co/docs/solutions/security/ai/attack-discovery#attack-discovery-rbac
* https://www.elastic.co/docs/solutions/security/get-started/automatic-migration

But our cumulative docs guidelines aren't clear about what the recommended pattern is. Do you know? The [applies-switch docs](https://elastic.github.io/docs-builder/syntax/applies-switch/) also don't cover this.